packages/devops/scripts/deployment/restoreE2eDataFromS3.sh

```diff
@@ -27,17 +28,23 @@ tar xf "$dump_file"
 
 echo "Disabling notification triggers"
 psql -U tupaia tupaia -c "ALTER TABLE survey_response DISABLE TRIGGER survey_response_trigger;"
```

Comment:
You could instead use `all` as per https://beyondessential.slack.com/archives/C0191QHH3QE/p1616128907004600

Reply:
Good idea, will do!

Reply:
Tried `psql -U tupaia tupaia -c "ALTER TABLE entity DISABLE TRIGGER all"` and got:

> ERROR:  permission denied: "RI_ConstraintTrigger_a_109476" is a system trigger

Reply:
Oh damn 😞

Reply:
Same in my local